src/current/v23.1/take-and-restore-encrypted-backups.md

```diff
@@ -163,6 +163,45 @@ BACKUP INTO 's3://{BUCKET NAME}?AWS_ACCESS_KEY_ID={KEY ID}&AWS_SECRET_ACCESS_KEY
     );
 ~~~
 
+#### Show a backup with multi-region encryption
+
+To view a backup with [multi-region encryption](#multi-region), use the `kms` option and the same KMS URIs that were used to create the backup:
```

Comment:
nit: this `#multi-region` link takes me to a somewhat random part of the docs page. Where did you intend it to go?

Reply:
nit: s/backup with multi-region encryption/a kms encrypted locality aware backup/r 

I'd replace this elsewhere on this page too.